.gitignore

```diff
@@ -13,3 +13,5 @@ test/common/
 test/images/
 *.pot
 POTFILES*
+tmp/*
+/src/test.jsx
```

Comment:
This looks like a debugging leftover. *Don't* add such local files to .gitignore, you are going to forget about them.